Modules/Product/Entities/Product.php

```diff
@@ -54,6 +58,28 @@ class Product extends Model
         'slug'
 
     ];
-}
 
+    public function productImages()
+    {
+        return $this->hasMany(ProductImage::class);
+    }
+    public function productVideos()
+    {
+        return $this->hasMany(Video::class);
+    }
+
+    public function productVariation()
```

Comment:
rename productVariation to productVariations. It should be plural as the relation is hasMany().